[CI] load test setup-environment failed on mainline

### Component(s)

testbed

### Describe the issue you're reporting

e.g. https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/13026633983/job/36336783485

```
Error: failed to update go.mod: go subcommand failed with args '[mod tidy -compat=1.22]': exit status 1, error message: go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/core/xidutils v0.118.0
go: github.com/open-telemetry/opentelemetry-collector-contrib/cmd/oteltestbedcol imports
	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sapmexporter imports
	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger imports
	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/core/xidutils: reading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/core/xidutils/go.mod at revision pkg/core/xidutils/v0.118.0: unknown revision pkg/core/xidutils/v0.118.0

make: *** [Makefile:359: genoteltestbedcol] Error 1
Error: Process completed with exit code 2.
```

--- a/cmd/oteltestbedcol/builder-config.yaml
+++ b/cmd/oteltestbedcol/builder-config.yaml
@@ -17,7 +17,7 @@ extensions:
 
 exporters:
   - gomod: go.opentelemetry.io/collector/exporter/debugexporter v0.118.1-0.20250123125445-24f88da7b583
-  - gomod: go.opentelemetry.io/collector/exporter/otlpexporter v0.118.1-0.20250123125445-24f88da7b583
+  - gomod: go.opentelemetry.io/collector/exporter/otlpexporter v0.119.0
   - gomod: go.opentelemetry.io/collector/exporter/otlphttpexporter v0.118.1-0.20250123125445-24f88da7b583
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/carbonexporter v0.119.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opencensusexporter v0.119.0

